tsconfig.json

```diff
@@ -1,23 +1,40 @@
 {
+  "exclude": [
+    "node_modules",
+    "lib",
+    "packages/*/*/node_modules",
```

Comment:
we don't need packages/*/** here. Assuming it would be removed with the general clean-up

Reply:
Yes sir, thanks! Copy-paste...